### DISCUSSION TITLE
How can I check, if RPI4 and create3 communicate over usb0 and not over WLAN?

### DISCUSSION THREAD

**Discussion by framlin** [iRobotEducation/create3_docs#278] [Questions] [answered]
Hello, I have connected a RPI4 and my new create3 via USB now.
It seems to work. I can ssh on the RPI4 and undock and dock the create3.
But how can I know, that the create3 and the RPI4 communicate over USB now and not still over WLAN?
I can ping 192168.1
Is there a monitoring-tool or something like that, that shows me such things?

Besides this, the create3 is a really really cool toy and I learned a lot about ROS2 playing around with it the last few days. Thank you very much.

**Comment by alsora** [comment, marked as answer]
You can use some network monitor tools.
For example: ROS 2 uses ports 7400, 7401, 7410, 7411 (as described here https://docs.ros.org/en/galactic/Concepts/About-Domain-ID.html#domain-id-to-udp-port-calculator).

You can use `tcpdump`:
```
sudo apt-get update && sudo apt-get install tcpdump
```

Then execute the command using:
```
sudo tcpdump -i any -X udp portrange 7401-7411
```

This should show you a log with the ROS messages and you should be able to see the network interface being used to transfer them.
